testsuite/cluster-test/src/suite.rs

```diff
@@ -68,4 +69,21 @@ impl ExperimentSuite {
         ));
         Self { experiments }
     }
+
+    fn new_land_blocking_suite(cluster: &Cluster) -> Self {
+        let mut experiments: Vec<Box<dyn Experiment>> = vec![];
+        experiments.push(Box::new(
+            PerformanceBenchmarkParams::new_nodes_down(0).build(cluster),
+        ));
+        Self { experiments }
+    }
+
+    pub fn new_by_name(cluster: &Cluster, name: &str) -> Result<Self> {
+        match name {
```

Comment:
maybe make suit name as const?

Reply:
My personal judgement here is to make strings const if they are used in multiple places (which is not the case here)